content/docs/command-reference/repro.md

```diff
@@ -10,21 +10,26 @@ analyzing dependencies and <abbr>outputs</abbr> of the target stages.
 ```usage
 usage: dvc repro [-h] [-q | -v] [-f] [-s] [-m] [--dry] [-i]
                  [-p] [-P] [-R] [--no-run-cache] [--force-downstream]
-                 [--no-commit] [--downstream] [--pull]
-                 [targets [targets ...]]
+                 [--no-commit] [--downstream] [--pull] [--glob]
+                 [targets [<target> ...]]
 
 positional arguments:
-  targets        Stage or path to dvc.yaml or .dvc file to reproduce. Using -R,
-                 directories to search for stages can also be given.
+  targets       Limit command scope to these .dvc or dvc.yaml files,
+                or stage names.
 ```
 
+> See [`targets`](#options) for more details.
+
 ## Description
 
 `dvc repro` provides a way to regenerate data pipeline results, by restoring the
 dependency graph (a [DAG](https://en.wikipedia.org/wiki/Directed_acyclic_graph))
 implicitly defined by the stages listed in `dvc.yaml`. The commands defined in
-these stages can then be executed in the correct order, reproducing pipeline
-results.
+these stages are then be executed in the correct order.
+
+For stages with multiple commands (having a list in the `cmd` field), commands
```

Comment:
not relevant?

Reply:
This is not being introduced here, it comes from a recent merge to master. Given some changes to the description this paragraph seems like it was moved which is why this part is green.

Reply:
p.s. there were merge conflicts I had to resolve which also cause some of these apparent changes + as you know our 80-line formatting tends to exaggerate the amount of changes.

Reply:
hmm, sorry, I still don't get it. If it comes from master, could you rebase/merge master? Otherwise this change adds a paragraph.

Reply:
Are we talking about the "For stages with multiple commands" paragraph? It was already there, look at old line 49: https://github.com/iterative/dvc.org/pull/1983/files#diff-6c1f3192f09e2722ba169e9fa219b3b5158bbafa470b382c2d6135db7aa1e20dL49

Reply:
But TBH I don't think I intended to move this one so I assume it has something to do with a merge conflict I had to solve during this PR. This is not about targets, it's about multiple commands.

Reply:
Reverted this move for now. Resolving